charts/trino/values.yaml

```diff
@@ -96,6 +96,66 @@ accessControl: {}
   #       ]
   #     }
 
+resourceGroups: {}
+  # # Resource groups file is mounted to /etc/trino/resource-groups/resource-groups.json
```

Comment:
nit: probably you can use smaller example

Reply:
This example matches the above access control example in terms of users and groups while being smaller than [Trino's documentation example](https://trino.io/docs/current/admin/resource-groups.html), to me it's appropriate